addons/website/views/website_templates.xml

```diff
@@ -142,6 +142,7 @@
             </script>
 
             <t t-raw="head or ''" name='layout_head'/>
+            <link rel="shortcut icon" type="image/x-icon" t-attf-href="data:image/x-icon;base64,#{website.favicon}"/>
```

Comment:
That will give terrible for performances and make very heavy pages. The favicon should be accessible from a /favicon route, so that browsers are able to cache it.